dlrover/python/common/global_context.py

```diff
@@ -208,3 +218,29 @@ def print_config(self):
     @property
     def user_id(self):
         return os.getenv(UserEnv.USER_ID, "")
+
+    def pre_check_enabled(self):
+        return (
+            self.pre_check_operators
+            and len(self.get_pre_check_operators()) > 0
+        )
+
+    def get_pre_check_operators(self):
+        result_ops = []
+        if not self.pre_check_operators:
+            return result_ops
+
+        for op_desc in self.pre_check_operators:
+            try:
```

Comment:
Why not check bypass here?

Reply:
Also for above question: `Since PRE_CEHCK_OPS and PRE_CHECK_BYPASS both include the complete operator package and name, it looks a little redundant here. How about only have PRE_CHECK_BYPASS and rename it to be PRE_CHECK_OPS?`

Good comment. A new way:

1. use 'PRE_CHECK_OPS' only
2. 'PRE_CHECK_OPS' 's format is {tuple2:bool}

- not use operator: no configuration at all  e.g. {}
- use operator but bypass: module and class configuration with 'False' value  e.g. {(m1, c1): False}
- use operator without bypass: module and class configuration with 'True' value  e.g. {(m1, c1): True}